hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/reconfig/TestOmReconfiguration.java

```diff
@@ -125,4 +131,59 @@ void unsetAllowListAllVolumes(String newValue) throws ReconfigurationException {
     assertEquals(OZONE_OM_VOLUME_LISTALL_ALLOWED_DEFAULT, cluster().getOzoneManager().getAllowListAllVolumes());
   }
 
+  @Test
+  void sstFilteringServiceInterval() throws ReconfigurationException {
+    // Tests reconfiguration of SST filtering service interval
+
+    // Get the original interval value
+    String originalValue = cluster().getOzoneManager().getConfiguration()
+        .get(OZONE_SNAPSHOT_SST_FILTERING_SERVICE_INTERVAL);
+    // Verify the original value is valid (should be larger than -1)
+    long originalInterval = cluster().getOzoneManager().getConfiguration()
+        .getTimeDuration(OZONE_SNAPSHOT_SST_FILTERING_SERVICE_INTERVAL,
+            org.apache.hadoop.ozone.om.OMConfigKeys.OZONE_SNAPSHOT_SST_FILTERING_SERVICE_INTERVAL_DEFAULT,
+            java.util.concurrent.TimeUnit.MILLISECONDS);
+    assertTrue(originalInterval > 0, "Original interval should be greater than 0");
```

Comment:
nit: for failure message that includes unexpected actual value, prefer `assertThat(originalInterval).isGreaterThan(...)`, or even `assertThat(originalInterval).isPositive()` in this case.

Reply:
Done. Thanks for the suggestion!